Entity/Repository/NotificationRepository.php

```diff
@@ -55,23 +42,55 @@ public function addRecipientConditions(QueryBuilder $builder, DoctrineRecipientI
             ->setParameter('class', ClassUtils::getClass($recipient))
             ->setParameter('id', $recipient->getId())
         ;
+
+        return $builder;
     }
 
     /**
-     * @param object $recipient
+     * @param DoctrineRecipientInterface  $recipient
      *
-     * @return Notification[]
+     * @return int
      */
-    public function countUndeliveredRecipientNotification($recipient)
+    public function countUndeliveredRecipientNotification(DoctrineRecipientInterface $recipient)
     {
-        $builder = $this->manager->createQueryBuilder();
+        $builder = $this->getEntityManager()->createQueryBuilder();
```

Comment:
Use `$this->createQueryBuilder('notification');` and remove `->from(Notification::class, 'notification')`